meshnets/utils/data_loading.py

```diff
@@ -1,17 +1,107 @@
-"""Load the mesh data"""
+""""Methods for loading mesh data"""
 
+from typing import Union
 from typing import Tuple
 
 from absl import logging
 import meshio
 import numpy as np
+import pyvista as pv
 
-# TODO(victor): modify the loader once a data format has been decided
 
+def load_edge_mesh_pv(
+    obj_path: str,
+    get_pressure: bool = True,
+    verbose: bool = False
+) -> Union[Tuple[np.ndarray, np.ndarray, np.ndarray], Tuple[np.ndarray,
+                                                            np.ndarray]]:
+    """Extract node coordinates, edges and pressure at nodes from a mesh file
+    using pyvista."""
 
-def load_mesh_from_obj(obj_path: str,
+    mesh = pv.read(obj_path)
+    # `use_all_points=False` discards duplicates and renumber the nodes
+    # TODO(victor): verify that node pressure array is modified accordingly
+    # WARNING: the remeshing should be done prior on the mesh file, not here
+    edge_mesh = mesh.extract_all_edges(use_all_points=True)
```

Comment:
It seems that with `use_all_points=True` also provides renumbering. Is this desired or a problem?

Reply:
This is unclear at the moment. I did not observe any renumbering with `use_all_points=True`, but I left a comment on the pyvista commit to ask for clarification of the method behavior. This will not be a problem for training and prediction, but if there is renumbering we risk to lose point correspondance between the original mesh and the extracted one.

Reply:
So I got in touch with a PyVista developer, which redirected me to VTK, and it seems that there is a small error in the pyvista docstring. So no point renumbering happens for `use_all_points=True` :)

Reply:
That is perfect Victor! Nicely done :)